terraform/environments/mlra/application_variables.json

```diff
@@ -12,6 +12,7 @@
       "maat_api_end_point": "http://maat-cd-api.aws.dev.legalservices.gov.uk/maatApi/laaStatus",
       "maat_db_url": "rds.maat.aws.dev.legalservices.gov.uk:1521:MAATDB",
       "maat_libra_wsdl_url": "http://infox.aws.dev.legalservices.gov.uk/infoX/gateway",
+      "app_google_analytics_4_tag_id": "G-KFYSC1V7RX",
```

Comment:
Mod platform is a public repository, is it OK to add Google tag ID?

Reply:
Fair point, best to keep the tag id in the AWS Parameter Store, have added it under the name MLRA_GOOGLE_ANALYTICS_4_TAG_ID

Reply:
@stone-dj please make sure to generate a new tag ID for storing in the parameter store, thanks